### ISSUE TITLE
Please add site https://www.wuxiax.com/novel/extreme-martial-arts-modifier.html

### ISSUE THREAD

**Issue by Okikaemmanuel1** [dteviot/WebToEpub#783]
Please note, I'm basically the only developer working on WebToEpub, and I'm not paid for doing this.  (WebToEpub is completely free, and generates no money.) By asking to add a site, you're asking me to give you some of my limited free time. So, I think it's not unreasonable for me to ask you to do as much as you can to help me.

### What settings did you use?  What didn't work?
* URL of first chapter
* CSS selector for element holding content to put into EPUB
* CSS selector for element holding Title of Chapter
* CSS selector for element(s) to remove

### If the Default Parser did not work, if you have developer skills, did you try writing a new parser?
Instructions https://dteviot.github.io/Projects/webToEpub_FAQ.html#write-parser

**Comment by dteviot** [comment]
@Okikaemmanuel1 
It's going to be a couple of weeks at least before I can get onto this.  
RL issues.

**Comment by dteviot** [comment]
@Okikaemmanuel1 
Support for that site was added in commit https://github.com/dteviot/WebToEpub/issues/730, dated 2022-05-10
This was available in release 0.0.0.149, which should be in the Chrome and Firefox stores.
I'm not sure why you're having problems with this.

I just tested https://www.wuxiax.com/novel/extreme-martial-arts-modifier.html, chapter 1 with the latest test version posted on Google drive and it seems to be working fine.
